share/module_bc.F

```diff
@@ -2615,6 +2615,113 @@ SUBROUTINE flow_dep_bdy_qnn  (  field,          &
 
    END SUBROUTINE flow_dep_bdy_qnn
 
+!-------------------------for ntu3m-------------------------------------------------
```

Comment:
@tzuchin12 
Can you briefly describe how this routine is different from the other flow dependent boundary conditions?

Reply:
Similarly for the a_ and tladj version

Reply:
Sorry, the a_ and the g_versions also. How are these routines different than the existing flow dependent boundary routines?

Reply:
The difference between the flow_dep_bdy_ntu and the other two existing routines is the allocated value to the field. The zero in flow_dep_bdy and one constant of ccn_conc in flow_dep_bdy_qnn are not suitable to be applied in the NTU scheme.